docs/source/training_eval.md

```diff
@@ -1,12 +1,23 @@
-
 # Training and Evaluation
-Many examples of data preparation, training and deployment of models using Transformers4Rec are available in our [examples](https://github.com/NVIDIA-Merlin/Transformers4Rec/tree/main/examples) directory.
 
-## Data loading
-Transformers4Rec leverages by default the NVTabular dataloader for GPU-accelerated loading of preprocessed data stored in Parquet format, which is a suitable format for being structured and queryable.
-The data in Parquet files are directly loaded to GPU memory as feature tensors. CPUs are also supported when GPUs are not available.
+```{contents}
+---
+depth: 2
+local: true
+backlinks: none
+---
+```
+
+## Data Loading
 
-The following example uses the NVTabular data loader, wrapped by the `DataLoader` that automatically sets some options from the dataset schema. Optionally the `PyarrowDataLoader` can also be used as a basic option, but it is slower and works only for small datasets, as the full data is loaded to CPU memory.
+By default, Transformers4Rec leverages the Merlin dataloader for GPU-accelerated loading of preprocessed data that is stored as Parquet files.
+Parquet enables this preprocessed data to be easily structured and queryable.
+The data in these parquet files are directly loaded to the GPU memory as feature tensors.
+CPUs are also supported when GPUs are not available.
+
+The following example uses the dataloader that is wrapped by the {class}`DataLoader <transformers4rec.torch.utils.data_utils.DataLoader>` class.
+The class automatically sets some options from the dataset schema.
+Optionally, you can use the {class}`PyarrowDataLoader <transformers4rec.torch.utils.data_utils.PyarrowDataLoader>` as a basic option, but it is slower and works only for small datasets since the full data is loaded to CPU memory.
 
 ```python
 train_loader = transformers4rec.torch.utils.data_utils.DataLoader.from_schema(
```

Comment:
T4Rec does not have a class `DataLoader`. It supports two classes `PyarrowDataLoader` and `NVTabularDataloader` ( renamed to `MerlinDataLoader` in #547)  
```suggestion
train_loader = transformers4rec.torch.utils.data_utils. MerlinDataLoader.from_schema(
```

Reply:
Changed to use `MerlinDataLoader`, but I had some trouble getting access to this class while poking around. Though, I clearly see it identified in the `data_utils.py` file.